sample_data/organizations.json

```diff
@@ -11,72 +11,16 @@
     "blockedUsers": [],
     "name": "The Unity Foundation",
     "description": "A foundation aimed at uniting the world and making it a better place for all.",
-    "location": "Delhi, India",
-    "isPublic": true,
-    "visibleInSearch": true,
-    "image": null,
-    "creator": "64378abd85008f171cf2990d",
-    "createdAt": "2023-04-13T05:16:52.827Z",
-    "__v": 0
-  },
-  {
-    "_id": "6537904485008f171cf29924",
-    "status": "ACTIVE",
-    "members": [
-      "65378abd85008f171cf2990d",
-      "67378abd85008f171cf2991d",
-      "67378abd85008f171cf2992d",
-      "67378abd85008f171cf2993d"
-    ],
-    "admins": ["65378abd85008f171cf2990d"],
-    "groupChats": [],
-    "posts": [],
-    "pinnedPosts": [],
-    "membershipRequests": [],
-    "blockedUsers": [],
-    "name": "Angel Foundation",
-    "description": "We are aimed at improving the education spaces for the under privileged girl child.",
-    "location": "Kolkata, West Bengal",
-    "isPublic": true,
-    "visibleInSearch": true,
-    "image": null,
-    "creator": "64378abd85008f171cf2990d",
-    "createdAt": "2023-04-13T05:16:52.827Z",
-    "__v": 0
-  },
-  {
-    "_id": "6637904485008f171cf29924",
-    "status": "ACTIVE",
-    "members": ["66378abd85008f171cf2990d"],
-    "admins": ["66378abd85008f171cf2990d"],
-    "groupChats": [],
-    "posts": [],
-    "pinnedPosts": [],
-    "membershipRequests": [],
-    "blockedUsers": [],
-    "name": "Hope Foundation",
-    "description": "A place where money and power doesn't matter. Free legal aid to everyone.",
-    "location": "Mountain View, CA",
-    "isPublic": true,
-    "visibleInSearch": true,
-    "image": null,
-    "creator": "64378abd85008f171cf2990d",
-    "createdAt": "2023-04-13T05:16:52.827Z",
-    "__v": 0
-  },
-  {
-    "_id": "6737904485008f171cf29924",
-    "status": "ACTIVE",
-    "members": ["67378abd85008f171cf2990d"],
-    "admins": ["67378abd85008f171cf2990d"],
-    "groupChats": [],
-    "posts": [],
-    "pinnedPosts": [],
-    "membershipRequests": [],
-    "blockedUsers": [],
-    "name": "Dignity Foundation",
-    "description": "A foundation aimed at improving the lives of old age citizens.",
-    "location": "Brooklyn, NYC",
+    "address": {
+      "city": "Delhi",
+      "countryCode": "IN",
+      "dependentLocality": "Some Dependent Locality",
+      "line1": "123 Random Street",
+      "line2": "Apartment 456",
+      "postalCode": "110001",
+      "sortingCode": "ABC-123",
+      "state": "Delhi"
+    },
```

Comment:
why were some smaple organizations deleted here?

Reply:
Actually Sir, I removed the sample organisations, and just kept one updating its located param to address just for testing purposes. I will bring back all the existing organisations before the PR merges.